gcloud-java-storage/src/main/java/com/google/gcloud/storage/BlobReadChannel.java

```diff
@@ -39,4 +39,11 @@
   void close();
 
   void seek(int position) throws IOException;
+
+  /**
+   * Sets the minimum size that will be read by a single RPC.
+   * Read data will be locally buffered until consumed.
+   */
+  void chunkSize(int chunkSize);
```

Comment:
In builders and other places I try to avoid the Bean's set style. Do you feel that specifying it (set prefix) here is more appropriate (not a builder and not immutable)?